src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -6,13 +6,15 @@
 import mate.academy.lib.Inject;
 import mate.academy.lib.Service;
 import mate.academy.model.User;
+import mate.academy.service.ShoppingCartService;
 import mate.academy.service.UserService;
 import mate.academy.util.HashUtil;
 
 @Service
 public class AuthenticationServiceImpl implements AuthenticationService {
     @Inject
     private UserService userService;
+    private ShoppingCartService shoppingCartService;
```

Comment:
```suggestion
    @Inject
    private ShoppingCartService shoppingCartService;
```
You lost annotation `@Inject`